_observing-your-data/metricsanalytics.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: default
+title: Metrics analytics
+nav_order: 40
+redirect_from:
+  - /observing-your-data/metricsanalytics/
+---
+
+# Metrics analytics
+Introduced 2.4
+{: .label .label-purple }
+
+With the release of OpenSearch 2.4, you can now ingest and visualize metrics data stored directly within OpenSearch using the **Metrics analytics** tool. This equips you with tools to analyze and correlate data across logs, traces, and metrics.
```

Comment:
Global: It should be "metric data" rather than "metrics data", but if "metric data" and "metrics" are synonymous, then we should just use "metrics" throughout.

Reply:
I accepted your edits to use metric